string robot_name
string planning_group

---

string[] named_target_poses
bool success